src/core/agent/services/identifierService.ts

```diff
@@ -198,7 +197,7 @@ class IdentifierService extends AgentService {
           displayName: identifier.prefix, //same as the id at the moment
           method: IdentifierType.KERI,
```

Comment:
This `method` needs to go from the metadata record, and we should remove the `IdentifierType` enum